wgsl/index.bs

```diff
@@ -2105,7 +2105,7 @@ Note: A [=constructible=] type has [=creation-fixed footprint=].
 
 The plain types with [=fixed footprint=] are any of:
 * a type with [=creation-fixed footprint=]
-* a [=fixed-size array=] type
+* a [=fixed-size array=] type (without further constraining its [=element count=])
 
 Note: The only valid use of a fixed-size array with an element count that is an
```

Comment:
See just below: we already have a note talking about the case that *doesn't* fall into the creation-fixed footprint case.